docs/requirements.txt

```diff
@@ -1,3 +1,6 @@
 sphinx_rtd_theme>=0.3.1
 pygments-lexer-solidity>=0.7.0
 sphinx-a4doc>=1.2.1
+
+# Sphinx 2.1.0 is the oldest version that accepts a lexer class in add_lexer()
+sphinx>=2.1.0
```

Comment:
Added this because [`b_docs` job is failing](https://app.circleci.com/pipelines/github/ethereum/solidity/16006/workflows/36518a98-e971-4340-9bf6-14e8e522456d/jobs/724319). Apparently there is an ancient version of Sphinx (v1.8.5) already installed in the image.